### PR TITLE
feat(bigquery): implement templates layouts with real content

### PR DESCRIPTION
**Pull request by kapoc3**
1. Implement layout for bigquery control panel 
imagen
2. Implement layout for bigquery promotional
imagen
3. Add constent for translate file from confluence
4.migrate submit button from formik to page
5. Refactor test with layout

### CODE REVIEW

File: src/i18n/en.js [hunk 1]
```diff
@@ -63,18 +63,34 @@ const messages_en = {
...
+    free_ul_item_strategy: `Get valuable information for your strategy.`,
+    plus_beginning: `Start`,
+    plus_big_query: `BigQuery`,
+    plus_body_step_one_MD: `Give access to BigQuery. [HELP] (https://cloud.google.com/bigquery/docs/?hl=es-419)`,
+    plus_body_step_two_MD: `Generate Doppler reports at Google Data Studio. [HELP](https://datastudio.google.com/overview)`,
+    plus_configuration: `Configuration`,
+    plus_control_panel: `Control Panel`,
+    plus_header_MD: `Get detailed reports to analyze the performance of your Email & Automation Marketing Campaigns.`,
+    plus_input_place_holder: `Add google account.`,
+    plus_message_remember: `Remember to Save Changes.`,
```
Comment (cbernat) [inline src/i18n/en.js:82]:
```suggestion
    plus_message_remember: `Remember to save changes.`,
```
We use only capitals for Doppler products.

File: src/i18n/en.js [hunk 2]
```diff
@@ -63,18 +63,34 @@ const messages_en = {
...
+    plus_message_saved: `Changes saved successfully.`,
+    plus_paragraph_two: `Google Big Query`,
+    plus_step_one: `Fisrt Step:`,
```
Comment (cbernat) [inline src/i18n/en.js:85]:
```suggestion
    plus_step_one: `Fisrt step:`,
```

File: src/i18n/es.js [hunk 3]
```diff
@@ -63,18 +63,34 @@ const messages_es = {
...
+    free_ul_item_strategy: `Obtener información valiosa para tu estrategia.`,
+    plus_beginning: `Inicio`,
+    plus_big_query: `BigQuery`,
+    plus_body_step_one_MD: `Otorgar acceso a BigQuery. [HELP](https://cloud.google.com/bigquery/docs/?hl=es-419)`,
+    plus_body_step_two_MD: `Generar reportes de Doppler en Google Data Studio. [HELP](https://datastudio.google.com/overview)`,
+    plus_configuration: `Configuración`,
+    plus_control_panel: `Panel de control`,
+    plus_header_MD: `Accede a reportes detallados para analizar el rendimiento de tus Campañas de Email & Automation Marketing.`,
+    plus_input_place_holder: `Agregar cuenta de google`,
+    plus_message_remember: `Recuerda Guardar los Cambios`,
```
Comment (cbernat) [inline src/i18n/es.js:82]:
```suggestion
    plus_message_remember: `Recuerda guardar los cambios`,
```

File: src/i18n/es.js [hunk 4]
```diff
@@ -63,18 +63,34 @@ const messages_es = {
...
+    plus_message_saved: `Cambios guardados con exito`,
```
Comment (cbernat) [inline src/i18n/es.js:83]:
```suggestion
    plus_message_saved: `Cambios guardados con éxito`,
```

File: src/i18n/es.js [hunk 5]
```diff
@@ -63,18 +63,34 @@ const messages_es = {
...
+    plus_paragraph_two: `Google Big Query`,
+    plus_step_one: `Primer Paso`,
```
Comment (cbernat) [inline src/i18n/es.js:85]:
```suggestion
    plus_step_one: `Primer paso`,
```

File: src/i18n/es.js [hunk 6]
```diff
@@ -63,18 +63,34 @@ const messages_es = {
...
+    plus_step_one_paragraph: `Los usuarios tendrán acceso a la información de tu cuenta almacenada en BigQuery. Recibirán una notificación por correo electrónico informando que les has dado dichos accesos.`,
+    plus_step_one_paragraph_MD: `**¡Psst!** Solo puedes otorgar acceso a **correos electrónicos de Google**, tanto de Gmail como de dominios empresariales.`,
+    plus_step_two: `Segundo Paso`,
```
Comment (cbernat) [inline src/i18n/es.js:88]:
```suggestion
    plus_step_two: `Segundo paso`,
```

File: src/i18n/en.js (hunk 1 above)
Comment (cbernat) [reply src/i18n/en.js:82]: @jmonacchi can you recheck these corrections? 🙏

File: src/i18n/en.js [hunk 7]
```diff
@@ -63,18 +63,34 @@ const messages_en = {
...
+    free_ul_item_insights: `Work from a customizable dashboard with editable views.`,
+    free_ul_item_strategy: `Get valuable information for your strategy.`,
+    plus_beginning: `Start`,
+    plus_big_query: `BigQuery`,
+    plus_body_step_one_MD: `Give access to BigQuery. [HELP] (https://cloud.google.com/bigquery/docs/?hl=es-419)`,
+    plus_body_step_two_MD: `Generate Doppler reports at Google Data Studio. [HELP](https://datastudio.google.com/overview)`,
+    plus_configuration: `Configuration`,
+    plus_control_panel: `Control Panel`,
+    plus_header_MD: `Get detailed reports to analyze the performance of your Email & Automation Marketing Campaigns.`,
+    plus_input_place_holder: `Add google account.`,
```
Comment (silsanchez8) [inline src/i18n/en.js:81]:
```suggestion
    plus_input_place_holder: `Add Google account.`,
```

File: src/i18n/en.js (hunk 2 above)
Comment (silsanchez8) [inline src/i18n/en.js:85]:
```suggestion
    plus_step_one: `First step:`,
```

File: src/i18n/es.js [hunk 8]
```diff
@@ -63,18 +63,34 @@ const messages_es = {
...
+    free_ul_item_insights: `Trabajar desde un tablero personalizado y con vistas editables.`,
+    free_ul_item_strategy: `Obtener información valiosa para tu estrategia.`,
+    plus_beginning: `Inicio`,
+    plus_big_query: `BigQuery`,
+    plus_body_step_one_MD: `Otorgar acceso a BigQuery. [HELP](https://cloud.google.com/bigquery/docs/?hl=es-419)`,
+    plus_body_step_two_MD: `Generar reportes de Doppler en Google Data Studio. [HELP](https://datastudio.google.com/overview)`,
+    plus_configuration: `Configuración`,
+    plus_control_panel: `Panel de control`,
+    plus_header_MD: `Accede a reportes detallados para analizar el rendimiento de tus Campañas de Email & Automation Marketing.`,
+    plus_input_place_holder: `Agregar cuenta de google`,
```
Comment (silsanchez8) [inline src/i18n/es.js:81]:
```suggestion
    plus_input_place_holder: `Agregar cuenta de Google`,
```

**Review by silsanchez8 — COMMENTED** [review]
Buenas! Deje algunos comentarios sobre revisión de contenidos 😉

File: src/i18n/en.js (hunk 7 above)
Comment (kapoc3) [reply src/i18n/en.js:81]: the item was removed because is unneccesary

File: src/i18n/en.js (hunk 2 above)
Comment (cbernat) [reply src/i18n/en.js:85]: First.

File: src/components/Integrations/BigQuery/AuthorizationPage.js [hunk 9]
```diff
@@ -59,23 +60,118 @@ const AuthorizationLayout = ({ dependencies: { bigQueryClient, dopplerUserApiCli
       <Promotional
```
Comment (jcamposmk) [inline src/components/Integrations/BigQuery/AuthorizationPage.js:60]: the promotional logo is very big. 
image

Comment (jcamposmk) [reply src/components/Integrations/BigQuery/AuthorizationPage.js:60]: promo page logo and gif can be moved to BigQuery folder

File: src/components/Integrations/BigQuery/AuthorizationPage.js [hunk 10]
```diff
@@ -59,23 +60,118 @@ const AuthorizationLayout = ({ dependencies: { bigQueryClient, dopplerUserApiCli
...
+      <div className="dp-container">
+        <div className="dp-rowflex">
+          <div className="col-sm-12 m-t-24">
+            <header className="m-b-30">
+              <a
+                href="index.html"
+                className="dp-button button-medium secondary-green button--round"
+              >
+                {' '}
+                Back{' '}
```
Comment (jcamposmk) [inline src/components/Integrations/BigQuery/AuthorizationPage.js:88]:
```suggestion
{_('common.back')}
```

File: src/components/Integrations/BigQuery/AuthorizationPage.js [hunk 11]
```diff
@@ -59,23 +60,118 @@ const AuthorizationLayout = ({ dependencies: { bigQueryClient, dopplerUserApiCli
...
+    <div data-active-menu="35" className="dp-library">
+      <div className="dp-container">
+        <div className="dp-rowflex">
+          <div className="col-sm-12 m-t-24">
+            <header className="m-b-30">
+              <a
+                href="index.html"
+                className="dp-button button-medium secondary-green button--round"
+              >
+                {' '}
```
Comment (jcamposmk) [inline src/components/Integrations/BigQuery/AuthorizationPage.js:87]: can you remove this line? it's not necessary

File: src/components/Integrations/BigQuery/AuthorizationPage.js [hunk 12]
```diff
@@ -59,23 +60,118 @@ const AuthorizationLayout = ({ dependencies: { bigQueryClient, dopplerUserApiCli
...
+          </div>
+          <span className="arrow"></span>
+        </div>
+      </header>
+      <section className="dp-container m-t-60">
+        <div className="dp-rowflex">
+          <div className="col-lg-6 col-md-12 col-sm-12 m-b-24 p-b-42">
+            <div className="dp-content-left">
+              <span>{_('big_query.plus_step_one')}</span>
+              <strong>
```
Comment (jcamposmk) [inline src/components/Integrations/BigQuery/AuthorizationPage.js:130]: Can you remove strong tag.? It's not necessary

File: src/components/Integrations/BigQuery/AuthorizationPage.js [hunk 13]
```diff
@@ -59,23 +60,118 @@ const AuthorizationLayout = ({ dependencies: { bigQueryClient, dopplerUserApiCli
...
+                <div className="p-heading">
+                  <FormattedMessageMarkdown id="big_query.plus_body_step_one_MD" />
+                </div>
+              </strong>
+              <p className="m-t-24">{_('big_query.plus_step_one_paragraph')}</p>
+              <div className="m-t-24">
+                <FormattedMessageMarkdown id="big_query.plus_step_one_paragraph_MD" />
+              </div>
+            </div>
+            <AuthorizationForm emails={data} onSubmit={onSubmit} />
```
Comment (jcamposmk) [inline src/components/Integrations/BigQuery/AuthorizationPage.js:140]: the form should go inside the dp-content-left class. That's why it looks like this:
image

File: src/components/Integrations/BigQuery/AuthorizationPage.js [hunk 14]
```diff
@@ -59,23 +60,118 @@ const AuthorizationLayout = ({ dependencies: { bigQueryClient, dopplerUserApiCli
...
+              </div>
+              <div className="dp-screen">
+                <img src={dataStudioGif} alt={_('big_query.free_alt_image')} />
+              </div>
+            </div>
+          </div>
+          <div className="col-sm-12 m-b-24">
+            <hr className="dp-h-divider"></hr>
+            <button type="button" className="dp-button button-medium primary-grey m-t-30 m-r-24">
+              {' '}
```
Comment (jcamposmk) [inline src/components/Integrations/BigQuery/AuthorizationPage.js:161]: can you remove this blank space ?

File: src/components/Integrations/BigQuery/AuthorizationPage.js [hunk 15]
```diff
@@ -59,23 +60,118 @@ const AuthorizationLayout = ({ dependencies: { bigQueryClient, dopplerUserApiCli
...
+              {_('common.cancel')}{' '}
```
Comment (jcamposmk) [inline src/components/Integrations/BigQuery/AuthorizationPage.js:162]: this one too ...

File: src/components/Integrations/BigQuery/AuthorizationPage.js [hunk 16]
```diff
@@ -59,23 +60,118 @@ const AuthorizationLayout = ({ dependencies: { bigQueryClient, dopplerUserApiCli
...
+            </button>
+            <button type="button">
+              <input
+                type="submit"
+                className="dp-button button-medium primary-green m-t-30"
+                form="big-query-configuration-form"
+                value={_('common.save')}
```
Comment (jcamposmk) [inline src/components/Integrations/BigQuery/AuthorizationPage.js:169]: When is clicked the submit button, the font color is grey. button states should be the same as the Submit Button component. Are you going to do this in another PR?
image

File: src/components/Integrations/BigQuery/ControlPanel/AuthorizationForm.js [hunk 17]
```diff
@@ -33,29 +32,33 @@ export const AuthorizationForm = ({ emails, onSubmit }) => {
...
-              />
-            )}
-          />
-        </fieldset>
-        <SubmitButton className="dp-button button-medium primary-green m-t-30">
-          {_('common.save')}
-        </SubmitButton>
-      </Form>
-    </Formik>
+    <div>
```
Comment (jcamposmk) [inline src/components/Integrations/BigQuery/ControlPanel/AuthorizationForm.js:35]: This wrapper `div` it's not necessary. Can you remove this `div` ?

File: src/components/Integrations/BigQuery/AuthorizationPage.js (hunk 10 above)
Comment (kapoc3) [reply src/components/Integrations/BigQuery/AuthorizationPage.js:88]: this code was removed because is unneccesary, the back button only used for UI library purpose

File: src/components/Integrations/BigQuery/AuthorizationPage.js (hunk 9 above)
Comment (jcamposmk) [reply src/components/Integrations/BigQuery/AuthorizationPage.js:60]: in figma you could download the logo with the original size

File: src/components/Integrations/BigQuery/AuthorizationPage.js (hunk 13 above)
Comment (kapoc3) [reply src/components/Integrations/BigQuery/AuthorizationPage.js]: suggest accepted and applied

File: src/components/Integrations/BigQuery/AuthorizationPage.js (hunk 16 above)
Comment (kapoc3) [reply src/components/Integrations/BigQuery/AuthorizationPage.js:153]: in next pr, ill resolve this with functionality of cancel button and user mesagges

File: src/i18n/en.js (hunk 7 above)
Comment (kapoc3) [reply src/i18n/en.js]: suggest accepted and applied

File: src/i18n/es.js [hunk 18]
```diff
@@ -59,22 +59,38 @@ const messages_es = {
...
+    free_title: `Conecta Doppler con BigQuery`,
+    free_ul_item_filter: `Aplicar diferentes filtros para enfocar el análisis en uno o varios aspectos de tu estrategia.`,
+    free_ul_item_insights: `Trabajar desde un tablero personalizado y con vistas editables.`,
+    free_ul_item_strategy: `Obtener información valiosa para tu estrategia.`,
+    plus_beginning: `Inicio`,
+    plus_big_query: `BigQuery`,
+    plus_body_step_one_MD: `Otorgar acceso a BigQuery. [HELP](https://cloud.google.com/bigquery/docs/?hl=es-419)`,
+    plus_body_step_two_MD: `Crear reportes. [HELP](https://datastudio.google.com/overview)`,
+    plus_configuration: `Configuración`,
+    plus_control_panel: `Panel de control`,
```
Comment (cbernat) [inline src/i18n/es.js:79]:
```suggestion
    plus_control_panel: `Panel de Control`,
```
image

File: src/i18n/es.js [hunk 19]
```diff
@@ -59,22 +59,38 @@ const messages_es = {
...
+    plus_header_MD: `Accede a reportes detallados para analizar el rendimiento de tus Campañas de Email & Automation Marketing.`,
+    plus_message_remember: `Recuerda guardar los cambios`,
+    plus_message_saved: `Cambios guardados con éxito`,
+    plus_paragraph_two: `Google Big Query`,
+    plus_step_one: `Primer paso`,
+    plus_step_one_paragraph: `Los usuarios tendrán acceso a la información de tu cuenta almacenada en BigQuery. Recibirán una notificación por correo electrónico informando que les has dado dichos accesos.`,
```
Comment (cbernat) [inline src/i18n/es.js:85]:
```suggestion
    plus_step_one_paragraph: `Los usuarios tendrán acceso a la información de tu cuenta almacenada en Big Query. Recibirán una notificación por correo electrónico informando que les has dado dichos accesos.`,
```
@kapoc3 Here I'm not sure if it is BigQuery or Big Query, but it should be the same in all the texts, right?

Comment (cbernat) [reply src/i18n/es.js:85]: Perhaps you could modify the only text with `Big Query` and that would be all. 
plus_paragraph_two: `Google BigQuery`

**Comment by cbernat** [comment]
With the text changes I think this is ready to ship 🚀

File: src/i18n/es.js (hunk 18 above)
Comment (kapoc3) [reply src/i18n/es.js]: suggest accepted and applied

File: src/components/Integrations/BigQuery/ControlPanel/AuthorizationForm.test.js [hunk 20]
```diff
@@ -90,17 +90,5 @@ describe('AuthorizationForm ', () => {
     // emails.length+2 because the second tag was added
     expect(cloudTags.querySelectorAll('li').length).toBe(emails.length + 2);
     expect(errors).not.toBeInTheDocument();
-    // simulate submit form
```
Comment (jcamposmk) [inline src/components/Integrations/BigQuery/ControlPanel/AuthorizationForm.test.js:93]: Can you remove the unused imports ? `waitFor` and `act`

File: src/components/Integrations/BigQuery/AuthorizationPage.js (hunk 9 above)
Comment (jcamposmk) [reply src/components/Integrations/BigQuery/AuthorizationPage.js:60]: you can move to outside bigquery_logo.png and google-data-studio.gif. The style folder must contain the styles, but is a empty folder. This folder should be deleted.
image
